docs/scos/dev/migration-program/migration-to-paas/paas-assessment-documents/is-s3-bucket-integrated-for-all-needed-storages-of-files.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Is S3 bucket integrated for all needed storages of files?
+description: This document allows you to assess if S3 bucket is integrated for all needed storages of files.
+template: howto-guide-template
+---
+
+Make sure that the project is not using any local storage, which may add more effort to migrate data to S3.
+
+Possible cases to check: Locally stored data, files stored in other places (not S3 bucket).
+
+Any third-party file storage is supposed to be migrated to S3 bucket usage. Files under the git repository are not counting here. In case the project uses third-party storage e.g. FTP server, it’s usage has to be replaced with an S3 bucket.
+
+You should be able to find this information in the prerequisites.
+
+## Resources for assessment
+
+* Backend
+* Optional: DevOps
```

Comment:
Is a DevOps needed to estimate the migration from a third-party FTP server?

Reply:
Looks like we can remove DevOps mentioning from this document